### PR TITLE
To-do-list: review

### PR DESCRIPTION
**Pull request by Debas-31**
# Review To-Do List
>In this project added:
- A project to-do list in a new feature branch.

### CODE REVIEW

File: .github/workflows/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (benjp009) [inline .github/workflows/.gitignore]: I can see that there is two .gitignore file, be carefull not to double the work and remove the wrong one

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+name: Linters
```
Comment (benjp009) [inline .github/workflows/linters.yml:1]: The linters files from the local machine shouldn't be pushed to live version

File: dist/index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>To Do List | Personal Software</title>
+<script defer src="main.js"></script></head>
+<body>
+    <main class="to-do-list-container">
+        <header>
+            <h3>Today's &nbsp; To Do<i class="fas fa-sync-alt"></i></h3>
```
Comment (benjp009) [inline dist/index.html:13]: Talking about Semantic, you should also be careful and use the H1 tag for the first title

File: src/data.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
+const listArray = [];
+/* eslint-disable no-use-before-define */
```
Comment (benjp009) [inline src/data.js:2]: Try to minimize the use of /* elint-disable */ in your code

**Review by benjp009 — APPROVED** [review]
# Congratulations🎉🎉 

Hi @Debas-31 , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional
_Check the comments under the review._ 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

changesrequested

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/.gitignore (hunk 1 above)
Comment (Debas-31) [reply .github/workflows/.gitignore]: - [ ] Thank you for the feedback, but I faced such issues in the first module. When I added the .gitignore file in the .github folder, it worked. Kindly do you have a tip on this?

File: .github/workflows/linters.yml (hunk 2 above)
Comment (Debas-31) [reply .github/workflows/linters.yml:1]: Thank you for the feedback. I will implement it.

File: dist/index.html (hunk 3 above)
Comment (Debas-31) [reply dist/index.html:13]: - [ ] Thank you for the feedback, but I have an opinion based on the project requirement it matters.

File: src/data.js (hunk 4 above)
Comment (Debas-31) [reply src/data.js:2]: Thank you i will do that

**Pull request merged**